app/demoservice/templates/demo_index.html

```diff
@@ -46,4 +53,18 @@ <h1>Running demos</h1>
       </div>
     </div>
   </div>
+  <script>
+    var states = document.getElementsByClassName("pr-state");
```

Comment:
Can you target a class which is prefixed with `js-` to separate the style classes from the functionality classes

Reply:
You could also use: `document.querySelectorAll('js-state');`